### DISCUSSION TITLE
Use both MicroG

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#197] [Polls]
It is quite possible that one day the error with unavailable content will reach everyone and they will need to use GmsCore (which is questionable, since some people still face the same problems, either with video freezing or with unavailable content), but until then some people do not see the point of changing the MicroG vendor. Should we bring back the old MicroG support as an option? What do you say?

**Comment by MtndaleRedneck** [comment]
Is there any benefit to sticking with MicroG over GmsCore? If not, eliminate it and make people bite the bullet and change now. They'll have to eventually. No reason to delay it.
